articles/azure-functions/functions-bindings-storage-blob-input.md

```diff
@@ -170,6 +170,9 @@ In the *function.json* file, the `queueTrigger` metadata property is used to spe
 
 The [configuration](#configuration) section explains these properties.
 
+> [!NOTE]
+> If you want to read binary data from the blob, make sure to specify `dataType` as `binary`. The default value for `dataType` is `string`, which means that data is interpreted as unicode.
```

Comment:
Actually, the `dataType` is [undefined](https://github.com/Azure/azure-functions-host/blob/7c81c0d416dc3402cbd3c023ca80e0778322d6ba/src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto#L361).

When DataType is undefined, it will use rich binding (e.g. `def main(input: func.InputStream)`). https://github.com/Azure/azure-functions-python-worker/tree/dev/tests/endtoend/blob_functions/blob_trigger

When DataType is 'string', it will use generic binding and cast the blob input type as string
(e.g. `def main(input: str):`). https://github.com/Azure/azure-functions-python-worker/tree/dev/tests/endtoend/blob_functions/get_blob_as_str

Same when DataType is 'binary', it will use generic binding and cast the blob to a bytes Python object
(e.g. `def main(input: bytes):`).
https://github.com/Azure/azure-functions-python-worker/tree/dev/tests/endtoend/blob_functions/get_blob_as_bytes

cc: @vrdmr @stefanushinardi